style/js/shield_defs.js

```diff
@@ -938,6 +938,120 @@ export function loadShields(shieldImages) {
   shields["US:WA:Spur"] = banneredShield(shields["US:WA"], ["SPUR"]);
   shields["US:WA:Business"] = banneredShield(shields["US:WA"], ["BUS"]);
   shields["US:WA:Alternate"] = banneredShield(shields["US:WA"], ["ALT"]);
+
+  shields["US:WI"] = {
+    backgroundImage: [
+      shieldImages.shield40_us_wi_2,
+      shieldImages.shield40_us_wi_3,
+    ],
+    textColor: "black",
+    padding: {
+      left: 3,
+      right: 3,
+      top: 3,
+      bottom: 5,
+    },
+  };
+  shields["US:WI:Business"] = banneredShield(shields["US:WI"], ["BUS"]);
+  shields["US:WI:Spur"] = banneredShield(shields["US:WI"], ["SPUR"]);
+  [
+    "Adams",
+    "Ashland",
+    "Barron",
+    "Bayfield",
+    "Brown",
+    "Buffalo",
+    "Burnett",
+    "Calumet",
+    "Chippewa",
+    "Clark",
+    "Columbia",
+    "Crawford",
+    "Dane",
+    "Dodge",
+    "Door",
+    "Douglas",
+    "Dunn",
+    "Eau Claire",
+    "Florence",
+    "Fond du Lac",
+    "Forest",
+    "Grant",
+    "Green",
+    "Green Lake",
+    "Iowa",
+    "Iron",
+    "Jackson",
+    "Jefferson",
+    "Juneau",
+    "Kenosha",
+    "Kewaunee",
+    "La Crosse",
+    "Lafayette",
+    "Langlade",
+    "Lincoln",
+    "Manitowoc",
+    "Marathon",
+    "Marinette",
+    "Marquette",
+    "Menominee",
+    "Milwaukee",
+    "Monroe",
+    "Oconto",
+    "Oneida",
+    "Outagamie",
+    "Ozaukee",
+    "Pepin",
+    "Pierce",
+    "Polk",
+    "Portage",
+    "Price",
+    "Racine",
+    "Richland",
+    "Rock",
+    "Rusk",
+    "Saint Croix",
+    "Sauk",
+    "Sawyer",
+    "Shawano",
+    "Sheboygan",
+    "Taylor",
+    "Trempealeau",
+    "Vernon",
+    "Vilas",
+    "Walworth",
+    "Washburn",
+    "Washington",
+    "Waukesha",
+    "Waupaca",
+    "Waushara",
+    "Winnebago",
+    "Wood",
+  ].forEach(
+    (county) =>
+      (shields[`US:WI:${county}`] = roundedRectShield(
+        "white",
+        "black",
+        "black",
+        1,
+        1
+      ))
+  );
+  shields["US:WI:Marquette:Truck"] = banneredShield(
+    shields["US:WI:Marquette"],
+    ["TRK"]
+  );
+  shields["US:WI:Rustic"] = {
+    backgroundImage: shieldImages.shield40_us_wi_rustic,
+    textColor: "#ffcd00",
```

Comment:
Interesting choice to superimpose the route number from the Rustic Road plaque onto the basic shape of the main route shield, versus reproducing only the plaque. This is consistent with the compromise taken in #226 for the Trans-Canada Highway in New Brunswick and would suggest a future treatment for U.S. Bicycle Routes. Out of curiosity, how do road maps of Wisconsin handle this shield?

Reply:
> Out of curiosity, how do road maps of Wisconsin handle this shield?

Not at all. Official state maps show circles for State Trunk Highways and rectangles for County Trunk Highways. Rustic Roads are unlabeled as such—the map just shows them at whatever other classification they may be.
![Screenshot from 2022-03-30 10-31-55](https://user-images.githubusercontent.com/1732117/160859780-459576a1-d76f-483f-8ebf-a7ea50122e29.png)

_From WisDOT's official map for [Walworth County (PDF)](https://wisconsindot.gov/Documents/travel/road/hwy-maps/county-maps/walworth.pdf)_